shell.nix

```diff
@@ -19,9 +11,5 @@ stdenv.mkDerivation {
 
   NIX_PATH = lib.concatStringsSep ":" [
     "nixpkgs=${pkgs.path}"
-    "nixpkgs-overlays=${overlay}"
-    "disciplina=https://github.com/DisciplinaOU/disciplina/archive/master.tar.gz"
-    "disciplina-faucet-frontend=https://github.com/DisciplinaOU/disciplina-faucet-frontend/archive/master.tar.gz"
-    "disciplina-explorer-frontend=https://github.com/DisciplinaOU/disciplina-explorer-frontend/archive/master.tar.gz"
```

Comment:
All of these serve a purpose: to allow per-deployment and deploy-time overrides, which is what we were planning to use for CD. Please restore them.